openapi.yml

```diff
@@ -112,3 +115,268 @@ paths:
         in: path
         required: true
         description: Version of the package
+components:
+  schemas:
+    IndexData:
```

Comment:
The schema names (`IndexData`, `Category`, etc) are based off a golang `struct` with the same name